__tests__/app.test.js

```diff
@@ -210,11 +190,46 @@ describe('/api/users', () =>{
   })
 })
 
+//ticket 15
+describe('/api/articles/:article_id/comments', () => {
+  describe('GET', ()=>{
+    test('200 - given article id responds with all comments for that article. Each comment should have properties: comment_id, votes, created_at, author, body', ()=> {
+      return request(app).get('/api/articles/1/comments')
+      .expect(200)
+      .then((res)=>{
+        const comments = res.body.comments;
+        expect(comments.length).toBe(11);
+        comments.forEach((comment)=>{
+          expect(comment.article_id).toBe(1);
+          expect(comments).not.toEqual([]);
+          expect.objectContaining({comment_id: expect.any(Number),
+          votes: expect.any(Number),
+        created_at: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String)
+      })
+        })
+      })
+    })
+    test('200 - article exists but has no comments', ()=>{
+      return request(app).get('/api/articles/2/comments')
+      .expect(200)
+      .then((res)=>{
+        expect(res.body.comments).toEqual([]);
+      })
+    })
+  })
+})
+
```

Comment:
Missing error handling here - what do we expect back if the article iD is valid but just not existent?

Reply:
is that not also covered by global 404?

Reply:
aah no it comes back as 200 currently